myapp/app/controllers/sessions_controller.rb

```diff
@@ -1,5 +1,6 @@
 class SessionsController < ApplicationController
   def new
+    redirect_to tasks_path, notice: 'Already logged in!' if logged_in?
```

Comment:
instead of checking in the method, how about creating a helper method and using in the `before_action`. 
or you can create a middlewhere and always redirect to next clouser if true.

Reply:
Fixed it in eb925c6